controllers/MappingsController.php

```diff
@@ -132,20 +131,21 @@ public function mappings() {
 			);
 		}
 		
-		header("Content-Type: application/json");
 		$json = Zotero_Utilities::formatJSON($json);
 		Z_Core::$MC->set($cacheKey, $json, $ttl);
-		
 		echo $json;
-		exit;
+		$this->end();
 	}
 	
 	
 	public function newItem() {
 		if (empty($_GET['itemType'])) {
 			$this->e400("'itemType' not provided");
 		}
-		
+
+		unset($this->queryParams['format']);
+		header("Content-Type: application/json");
```

Comment:
Can we just set 'json' as the default `format` for the mappings actions for v1/v2 in API.inc.php?

Reply:
Right. That is definitely cleaner